File: Selenium/Activities/activity6.java
package Activities;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.Keys;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.interactions.Actions;

import javax.swing.*;

public class activity6 {
    public static void main(String[] args) {
        System.setProperty(FirefoxDriver.SystemProperty.BROWSER_LOGFILE, "/dev/null");
        WebDriverManager.firefoxdriver().setup();
        WebDriver driver = new FirefoxDriver();
        driver.get("https://www.training-support.net/selenium/input-events");
        Actions text = new Actions(driver);
        driver.findElement(By.id("keyPressNotify")).click();
        text.sendKeys("K").build().perform();
        text.keyDown(Keys.CONTROL).sendKeys("a").sendKeys("c").keyUp(Keys.CONTROL).build().perform();
        System.out.println();
    }}

File: Selenium/Activities/activity6.py
from selenium import webdriver
from selenium.webdriver import ActionChains, Keys
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager
service = FirefoxService(GeckoDriverManager().install())



with webdriver.Firefox(service=service) as driver:
    driver.get("https://www.training-support.net/selenium/input-events")


    print("Page title is: ", driver.title)


    actions = ActionChains(driver)


    actions.send_keys("A")
    actions.key_down(Keys.CONTROL)
    actions.send_keys("a")
    actions.send_keys("c") 
    actions.key_up(Keys.CONTROL)
    actions.perform()


    actions.send_keys("A").key_down(Keys.CONTROL).send_keys("a").send_keys("c").key_up(Keys.CONTROL).perform()